grass7/raster/r.basin/r.basin.py

```diff
@@ -465,9 +465,7 @@ def main():
         # Rectangle containing basin
         basin_east = baricenter_slope_baricenter[33]
         basin_north = baricenter_slope_baricenter[34]
-        info_region_basin = grass.read_command(
-            "g.region", vect=options["prefix"] + "_" + mapname[0] + "_basin", flags="m"
-        )
+        info_region_basin = grass.read_command("g.region", rast=r_basin, flags="m")
```

Comment:
```suggestion
        info_region_basin = grass.read_command("g.region", raster=r_basin, flags="m")
```

(complete parameter names are preferred)